src/Root.js

```diff
@@ -1,24 +1,31 @@
 import * as React from 'react'
-import { Route, HashRouter } from 'react-router-dom'
-import { Switch } from 'react-router-dom'
-
+import * as ReactRedux from 'react-redux'
+import * as ReactReduxFirebase from 'react-redux-firebase'
+import * as Redux from 'redux'
+import { Route, HashRouter, Switch } from 'react-router-dom'
+import PropTypes from 'prop-types'
 import MenuAppBar from './components/menu-app-bar'
 import Inventory from './components/inventory'
 import SalesReport from './components/sales-report'
 import Add from './components/add'
 import Payment from './components/payment'
-
 import CONFIG from './constants/config'
 import ROUTE from './constants/route'
 import SignIn from './components/auth/SignIn'
 import SignUp from './components/auth/SignUp'
 import Orders from './components/orders'
+import orderAction from './redux/actions/order'
+import selectors from './redux/selectors'
 
 const styles = {
   backgroundColor: 'cornflowerblue',
 }
 
 class Root extends React.Component {
+  componentDidMount() {
+    this.props.orderInitialize()
```

Comment:
I do not agree with that. This is ```orderInitialize``` )(initialize function in order actions) and if I rename it will be not clear what we initialize. I also have ```walletInitialize```